sigle/src/ui/Form.ts

```diff
@@ -0,0 +1,89 @@
+import { styled } from '../stitches.config';
+
+export const FormRow = styled('div', {
+  mb: '$5',
+});
+
+export const FormLabel = styled('label', {
+  width: '100%',
+  display: 'block',
+  fontSize: '$2',
+  color: '$gray11',
+  mb: '$3',
+});
+
+export const FormInput = styled('input', {
+  '&[type]': {
+    appearance: 'none',
+    borderWidth: '0',
+    boxSizing: 'border-box',
+    fontFamily: 'inherit',
+    margin: '0',
+    outline: 'none',
+    padding: '0',
+    width: '100%',
+    WebkitTapHighlightColor: 'rgba(0,0,0,0)',
+    backgroundColor: '$gray3',
+    boxShadow: '0 0 0 1px $colors$gray7',
+    borderRadius: '$1',
+    px: '$2',
+    py: '$1',
+    fontSize: '$1',
+    color: '$gray11',
+
+    '&:focus': {
+      boxShadow: '0 0 0 2px $colors$gray8',
+    },
+
+    '&::placeholder': {
+      color: '$gray9',
+    },
+  },
+
+  '&[type="date"]::-webkit-calendar-picker-indicator': {
+    background: 'url(/static/img/Calendar.svg) no-repeat',
```

Comment:
You used an SVG as it was not possible to import a radix icon from the npm package, correct?

Reply:
Yes, that's correct!